src/ripple/app/main/Application.cpp

```diff
@@ -1566,8 +1566,19 @@ bool ApplicationImp::setup()
 
         Resource::Charge loadType = Resource::feeReferenceRPC;
         Resource::Consumer c;
-        RPC::Context context { journal ("RPCHandler"), jvCommand, *this,
-            loadType, getOPs (), getLedgerMaster(), c, Role::ADMIN };
+        RPC::Context context {
+            journal ("RPCHandler"),
+            jvCommand,
+            *this,
+            loadType,
+            getOPs (),
+            getLedgerMaster (),
+            c,
+            Role::ADMIN,
+            {},
+            {},
+            {{},{}}
```

Comment:
What was the warning that motivated the addition of these last three member initializations?

Reply:
`-Wmissing-field-initializers` (clang). If you'd prefer I disable that one, I'm happy to do that.

Reply:
Thanks.  What do you think about silencing them in Context.h instead, like this:
```
struct Context
{
    // ...
    std::shared_ptr<JobQueue::Coro> coro{};
    InfoSub::pointer infoSub{};
    Headers headers{};
};
```
It isn't a huge deal because we only construct like this in one place.  But when I read the code in Application.cpp, I had to look up in Context.h to see what it was actually doing.  If we do it in Context.h, it seems easier to read to me.

Reply:
yep - I like that better. will change.

Reply:
Thanks.